README.adoc

```diff
@@ -1,3 +1,23 @@
+=== ⚠️ CAUTION
+
+The developer team is currently preparing the release of OCaml 5.0. This release
+sports a full rewrite of its runtime system for shared-memory parallel
+programming using domains and native support for concurrent programming using
+effect handlers.
+
+The initial release of OCaml 5.0 will only support the native compiler under
+ARM64 and x86-64 architectures under Linux and macOS. On Windows, only the
+Mingw-w64 port is supported.  Support for other 64-bit architectures and systems
+will be added back in later releases. On 32-bit systems, only the bytecode
+compiler is supported.  Native-code support for these systems is under
+discussion.
+
+Owing to the large number of changes, the initial 5.0 release will be more
+experimental than usual.  It is recommended that all users wanting a stable
+release use the 4.14 release which will continue to be supported and updated
+while 5.0 reaches feature and stability parity. Similarly, if you need one of
+the ports not yet supported in the 5.0 release you must use the 4.14 release.
```

Comment:
I would switch the ordering of the second and third paragraph. The currently-third paragraph is more important for users.

Reply:
Makes sense, fixed.